src/wp-includes/js/autosave.js

```diff
@@ -571,10 +779,19 @@ window.autosave = function() {
 			};
 		}
 
-		// Wait for TinyMCE to initialize plus 1 sec. for any external css to finish loading,
-		// then 'save' to the textarea before setting initialCompareString.
-		// This avoids any insignificant differences between the initial textarea content and the content
-		// extracted from the editor.
+
+		/**
+		 * @summary Sets the autosave time out.
+		 *
+		 * Wait for TinyMCE to initialize plus 1 sec. for any external css to finish loading,
```

Comment:
Rephrase 'sec.' to 'second' to improve readability (because of the period it now it looks like a sentence end)